library/Icingadb/Widget/StateChange.php

```diff
@@ -55,9 +58,15 @@ public function setCurrentStateBallSize(string $size): self
         return $this;
     }
 
+    public function isReversed(): bool
```

Comment:
Please drop this and add the icon here directly. The state knows which icon to use now, so use it please.

Reply:
State (Model) does not exist here.

Reply:
uhm, okay, but please get rid of this method. That's an implementation detail no-one should care about. Consider a `setIcon` method for example.